conductor-clients/java/conductor-java-sdk/conductor-client/src/main/java/com/netflix/conductor/common/metadata/tasks/TaskDef.java

```diff
@@ -47,6 +47,8 @@ public enum RetryLogic {
 
```

Comment:
@shaileshpadave anything that's under `conductor-clients/java/conductor-java-sdk/` is part of the Java Client v4 project.

Let's keep the conductor core changes and client changes in separate PRs

Reply:
ACK